content/en/05-reference/01-admin/09-user-mgmt.md

```diff
@@ -0,0 +1,107 @@
+metaDescription: "Share and manage access to your Pixie data."
+order: 6
+redirect_from:
+    - /admin/permissions/
+---
+
+<Alert variant="outlined" severity="info">This information applies to Pixie instances that use Auth0 for Authentication. <a href="https://docs.px.dev/installing-pixie/install-guides/community-cloud-for-pixie/">Community Cloud for Pixie</a> ships with Auth0. <a href="https://docs.px.dev/installing-pixie/install-guides/self-hosted-pixie/">Self-Hosted Pixie</a> must be <a href="https://docs.px.dev/reference/admin/authentication/#enabling-auth0">manually configured</a> to use Auth0. To learn how to invite users to Self-Hosted Pixie instances using open source authentication, see the Quick Start <a href="https://docs.px.dev/installing-pixie/install-guides/self-hosted-pixie/#invite-others-to-your-organization-(optional)">instructions</a>.
+</Alert>
+
+Pixie has features to share and manage access your Pixie organization:
+
+- [Invite users](/reference/admin/user-mgmt#invite-users)
+- [Enable approvals for new users](/reference/admin/user-mgmt#enable-approvals-for-new-users)
+- [Disable existing invite links](/reference/admin/user-mgmt#disable-invite-links)
+- [Remove users](/reference/admin/user-mgmt#remove-users)
+
+Users added to your organization can view Pixie live views, query running clusters, and deploy new Pixie clusters.
+
+## Invite Users
+
+### GSuite Organizations
+
+Pixie handles GSuite orgs differently than other orgs.
+
+You are part of a Pixie GSuite org if:
+
+1. The original creator of the org signs into Pixie using the "Sign in with Google" button AND
+2. The email the admin uses to sign in is part of a GSuite account.
+
+GSuite orgs can invite two types of users:
+
+- **Teammates with GSuite emails**: should use the "Sign-Up with Google" button on the [Sign up](https://work.withpixie.ai/auth/signup) page. GSuite emails will automatically join the Pixie org corresponding to their GSuite domain. _DO NOT use Invite Links to invite teammates with GSuite emails._ For greater control, enable [User Approvals](/reference/admin/user-mgmt#enable-approvals-for-new-users).
+
+- **Teammates using Gmail or creating custom email/password Pixie accounts**: should be sent [Invite Links](/reference/admin/user-mgmt#invite-links).
+
+### All Other Organizations
+
+If your organization is not a [GSuite organization](/reference/admin/user-mgmt#gsuite-organnizations), you can invite users with [Invite Links](/reference/admin/user-mgmt#invite-links).
+
+<Alert variant="outlined" severity="warning">Invited teammates using GSuite emails MUST sign up for Pixie using the "Sign-Up with Email" button. If a teammate accidentally signs up for Pixie using SSO ("Sign-Up with Google" button), they will not be able to join your Org. To fix this, they will need to sign up for Pixie again using the "Sign-Up with Email" button with a new or modified email (e.g. adding `+pixie` to their email, like anna+pixie@fakeemail.com)</Alert>
```

Comment:
@aimichelle  - I understand that we have these requirements for good reason and not challenging the requirements themselves, but I wonder if there is a way to augment the flows to add guardrails/etc during the invite link or signup process to make falling into the "bad" cases less easy to do